cpp/include/Ice/LocalExceptions.h

```diff
@@ -91,18 +83,12 @@ namespace Ice
             std::string facet,
             std::string operation);
 
-        /**
-         * Constructs a RequestFailedException without specifying the details of the current request. The details will
-         * be filled-in automatically by the Ice runtime before marshaling the exception.
-         * @param file The file where this exception is constructed. This C string is not copied.
-         * @param line The line where this exception is constructed.
-         * @param replyStatus The reply status.
-         */
```

Comment:
Even if we don't want generate anything for this, why remove this comment and the one above?
It's still useful to people reading the code themselves. Especially for not-obvious details like
"The details will be filled-in automatically by the Ice runtime..."